kusto/kusto_examples_test.go

```diff
@@ -26,11 +26,8 @@ func Example_simple() {
 		CollectionTime time.Time
 	}
 
-	authorizer := Authorization{
-		Config: auth.NewClientCredentialsConfig("clientID", "clientSecret", "tenantID"),
-	}
-
-	client, err := New("endpoint", authorizer)
+	kcsb := GetConnectionStringBuilder("endpoint;tenantid=tenentID;appclientid=clientID;appkey=clientSecret")
```

Comment:
I think the correct way to build a csb is with the builder methods, not to parse one